docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/duo-source.md

```diff
@@ -23,11 +23,11 @@ healthy and collecting by utilizing Health Events.
 
 A Duo Source goes through the following states when created:
 
-1. **Pending**: Once the Source is submitted it is validated, stored, and placed in a **Pending** state.
-1. **Started**: A collection task is created on the Hosted Collector.
-1. **Initialized**: The task configuration is complete in Sumo Logic.
-1. **Authenticated**: The Source successfully authenticated with Duo.
-1. **Collecting**: The Source is actively collecting data from Duo.
+1. **Pending**. Once the Source is submitted it is validated, stored, and placed in a **Pending** state.
+1. **Started**. A collection task is created on the Hosted Collector.
+1. **Initialized**. The task configuration is complete in Sumo Logic.
+1. **Authenticated**. The Source successfully authenticated with Duo.
+1. **Collecting**. The Source is actively collecting data from Duo.
```

Comment:
Hi @JainM6 , I've unresolved this comment so that I can address Carlos' query. 

@carlos-castillo-jask , I believe she made these changes because the same ones in a related PR were requested by @stacykor (see **https://github.com/SumoLogic/sumologic-documentation/pull/994**).